commit ca68ee1279770d44e85df49e7193afe569caae49
Author: Thomas Falcon <tlfalcon@linux.vnet.ibm.com>
Date:   Fri Mar 17 00:48:25 2017 +0000

    ibmveth: calculate gso_segs for large packets
    
    [ Upstream commit 94acf164dc8f1184e8d0737be7125134c2701dbe ]
    
    Include calculations to compute the number of segments
    that comprise an aggregated large packet.
    
    Signed-off-by: Thomas Falcon <tlfalcon@linux.vnet.ibm.com>
    Reviewed-by: Marcelo Ricardo Leitner <marcelo.leitner@gmail.com>
    Reviewed-by: Jonathan Maxwell <jmaxwell37@gmail.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1431_ca68ee1279770d44e85df49e7193afe569caae49.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1431_ca68ee1279770d44e85df49e7193afe569caae49.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
